sos/utilities.py

```diff
@@ -124,6 +125,12 @@ def _child_prep_fn():
         if (chdir):
             os.chdir(chdir)
 
+    def _check_poller(proc):
+        if poller():
+            proc.terminate()
+            raise SoSTimeoutError
+        time.sleep(0.1)
```

Comment:
We had `time.sleep(0.01)`, some reason to increase this? (but I think even the increase does not matter).

Reply:
I ran a couple tests with this while re-writing this section. The TL;DR of it is using `0.1` gives us lower CPU consumption, however at the cost of runtime. This makes sense as we're doing 10x less calls to `poller()`, however we stack up the extra fractions of a second across all the hundreds/thousands of commands we run.

Initially, my opinion is/was that this is preferable - however, I'm now thinking of this in the context of nodes with extremely high numbers of namespaces - which would definitely cause the runtime to increase dramatically more than "normal" systems.

Reverting this back to 0.01 for now, and I'll make a note to do some more investigation on this.